pro/src/ui-kit/index.ts

```diff
@@ -3,3 +3,4 @@ export { default as Divider } from './Divider'
 export { default as SubmitButton } from './SubmitButton'
 export { default as Tag } from './Tag'
 export * from './form'
+export { default as Typography } from './typography'
```

Comment:
je voyais typography un peu comme une catégorie, comme form, plutôt. 
```suggestion
export * from './typography'
```